### PR TITLE
Move the details from CONTRIBUTING.md to the manual #8734

### PR DESCRIPTION
**Pull request by cillianderoiste**

### CODE REVIEW

**Comment by lethalman** [comment]
:+1: The section seems to "up" in the list to me, maybe below "Coding conventions" is better.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,72 +7,6 @@
...
-* Hydra builds for master and staging should not be used as testing platform, it's a build farm for changes that have been already tested.
-* Master should only see non-breaking commits that do not cause mass rebuilds.
-* Staging should only see non-breaking mass-rebuild commits. That means it's not to be used for testing, and changes must have been well tested already. [Read policy here](http://comments.gmane.org/gmane.linux.distributions.nixos/13447).
-* If staging is already in a broken state, please refrain from adding extra new breakages. Stabilize it for a few days, merge into master, then resume development on staging. [Keep an eye on the staging evaluations here](http://hydra.nixos.org/jobset/nixpkgs/staging#tabs-evaluations).
-* When changing the bootloader installation process, extra care must be taken. Grub installations cannot be rolled back, hence changes may break people's installations forever. For any non-trivial change to the bootloader please file a PR asking for review, especially from @edolstra.
+Se the nixpkgs manual for details on how to [Submit changes to nixpkgs](http://nixos.org/nixpkgs/manual/#chap-submitting-changes).
```
Comment (domenkozar) [inline CONTRIBUTING.md]: s/Se/See/

**Comment by cillianderoiste** [comment]
@lethalman sounds good to me, I've moved it down. I've also fixed the typo @domenkozar.

**Comment by jagajaga** [comment]
I don't like the idea. The chances that an additional link will be opened less than just open `CONTRIBUTING.md`.

**Comment by cillianderoiste** [comment]
@jagajaga I agree that it's not ideal, but currently we have similar contribution guidelines in a few places. I think there are several articles in the wiki and this CONTRIBUTING.md file was added relatively recently and has been growing. If someone searches the internet for such documentation, they could end up in very different places. IMO it's better to point people to one definitive place for such documentation.

The link to CONTRIBUTING.md is displayed by github when you create a new issue, so I thought it made sense to keep the section at the top which provides some tips on creating issues, but for further details to point them to the manual. There is much more information there already which would also be of interest.

**Comment by lethalman** [comment]
Can we change the reference to the manual to `http://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/download-by-type/doc/manual` ?

**Comment by lethalman** [comment]
Merged I'm changing the link manually. Thanks.

**Comment by jagajaga** [comment]
Bad idea.

**Pull request merged**